lib/missing_step_error.ex

```diff
@@ -0,0 +1,62 @@
+defmodule MissingStepError do
+  @moduledoc """
+  Raises an error, because a feature step is missing its implementation.
+
+  The message of the error will give the user a useful code snippet where
+  variables in feature steps are converted to regex capture groups.
+  """
+
+  defexception [:message]
+
+  @number_regex ~r/(^|\s)\d+(\s|$)/
+  @single_quote_regex ~r/'[^']+'/
+  @double_quote_regex ~r/"[^"]+"/
+
+  def exception(step_text: step_text, step_type: step_type) do
```

Comment:
What I have done here is define `exception/1` taking a Keyword list and returning a struct with just one element: `message`.

I am relying on the default implementation of `message/1` which just pulls out `:message` from the struct and returns that.